classes/webservice/WebserviceSpecificManagementAttachments.php

```diff
@@ -43,7 +43,7 @@ class WebserviceSpecificManagementAttachmentsCore implements WebserviceSpecificM
     protected $objOutput;
 
     /**
-     * @var WebserviceRequest
+     * @var WebserviceRequestCore
```

Comment:
Same here, I'm not sure we should rely on the `*Core` classes? 🤔

Reply:
I agree the `*Core` classes should never be used they are part of the legacy PrestaShop override system but are not meant to be used like this If PHPStan is complaining about this we need to find a configuration to make it understand accept or ignore those cases

Reply:
Sorry but at the l.178 and in the Interface, it's not the object but *Core which is defined. If we want to change the interface, we need to do it in an another PR. 

Ping @atomiix & @jolelievre

Reply:
Why not modify the interface in this PR then? It has no impact since `WebserviceRequest` and `WebserviceRequestCore` are nearly synonyms, I agree this can be considered a BC break but I think we can assume this one with some peace of mind!

Reply:
Because it's not in the scope of the PR to create BC Breaks. The scope is having a "PHPStan Level 3 on classes/".

Reply:
Done.